paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -465,6 +465,12 @@ def downscale_aws_resource(self, filtered_slaves, current_capacity, target_capac
                         "This is a SFR so we must kill at least one slave to prevent the autoscaler "
                         "getting stuck whilst scaling down gradually",
                     )
+                    if new_capacity < 1 and self.sfr['SpotFleetRequestState'] == 'active':
+                        self.log.info(
+                            "However, this is an active SFR and we can't set capacity less than 1 "
```

Comment:
can you make this more precise? 'can't set capacity to less than 1; no action taken for this SFR'.